src/encoded/static/components/browse/components/above-table-controls/SelectedFilesDownloadButton.js

```diff
@@ -207,22 +207,33 @@ class SelectedFilesDownloadModal extends React.PureComponent {
                 </Modal.Header>
 
                 <Modal.Body>
-
+                    <div className="important-notes-section">
+                        <h4 className="mb-07 text-500">Important</h4>
+                        <ul className="mb-25">
+                            <li className="mb-05">
+                                <span className="text-danger"><b>As of December 1, 2020</b>, you must include an <b>access key</b> in your cURL command for bulk downloads.</span>
+                            </li>
+                            <li className="mb-05">You can configure the access key in {isSignedIn ? <a href={profileHref} target="_blank" rel="noopener noreferrer">your profile</a> : 'your profile'}, then use it in place of <em>{'<access_key_id>:<access_key_secret>'}</em>, below.</li>
+                            {!isSignedIn ?
+                                <li>If you don’t already have an account, you can log in with your Google or GitHub credentials.</li>
+                                : null}
+                        </ul>
+                    </div>
                     <p>Please press the &quot;Download&quot; button below to save the metadata TSV file which contains download URLs and other information for the selected files.</p>
                     <p>Once you have saved the metadata TSV, you may download the files on any machine or server with the following cURL command:</p>
-                    <ModalCodeSnippet filename={suggestedFilename} isSignedIn={isSignedIn} />
+                    <ModalCodeSnippet filename={suggestedFilename} isSignedIn />
 
                     { isSignedIn || foundUnpublishedFiles ?
                         <div className="extra-notes-section">
                             <h4 className="mt-2 mb-07 text-500">Notes</h4>
                             <ul className="mb-25">
-                                { isSignedIn ?
+                                {/* { isSignedIn ?
```

Comment:
I would maybe keep some variation of this text ("To download files which are not yet released..."), but without the `{ isSignedIn ?` & `: null }` surrounding it, but unsure / would double-check w. Burak.

Reply:
Also, since we get `session` passed into here now, we can pretty much just find+replace uses of `isSignedIn` with `session` I think to cleanup code a tiny bit. This line (`const isSignedIn = !!(userInfo && userInfo.details && userInfo.details.email && userInfo.id_token);`) was originally just a quick workaround to having to pass down session down into here (though passing down `session` I think is better).